src/reflect/scala/reflect/internal/transform/UnCurry.scala

```diff
@@ -77,6 +101,60 @@ trait UnCurry {
     }
   }
 
+  private def varargForwarderSym(currentClass: Symbol, origSym: Symbol, newInfo: Type): Symbol = {
+    val forwSym = currentClass.newMethod(origSym.name.toTermName, origSym.pos, VARARGS | SYNTHETIC | origSym.flags & ~DEFERRED)
```

Comment:
Thanks, I'll have a look. I'm a bit surprised about the failing test, since this seemed to pass on my system. Also, there are some compiler crashes, weird... :-/